internal/build/image_builder_test.go

```diff
@@ -183,13 +183,13 @@ func TestCleanUpBuildKitErrors(t *testing.T) {
 	}{
 		// actual error currently emitted by buildkit when a `RUN` fails
 		{
-			buildKitError:     "failed to solve with frontend dockerfile.v0: failed to build LLB: executor failed running [/bin/sh -c go install github.com/windmilleng/servantes/vigoda]: runc did not terminate sucessfully",
```

Comment:
I think these are typos that exist upstream in runc: https://github.com/search?q=%22terminate+sucessfully%22&type=Code so they should remain here as is